### ISSUE TITLE
Training my own database

### ISSUE THREAD

**Issue by Chen65010445** [vanhuyz/CycleGAN-TensorFlow#46]
What should I do to train my own database? Just replace the pictures in data folder or I should do something else?

**Comment by vanhuyz** [comment]
Yes, just replace the pictures in data folder. Check `$ python3 build_data.py --help` to make sure how to specify the data folder :slightly_smiling_face:
